suite-common/bluetooth/src/filterOutOldDuplicatesBySameProps.ts

```diff
@@ -0,0 +1,19 @@
+import { BluetoothDeviceCommon } from './types';
+
+// There is a UX need to remap devices that are in pairing mode and emit more than once
+// we do this by filtering out duplicates by name, color and model
+export const filterOutOldDuplicatesBySameProps = <T extends BluetoothDeviceCommon>(devices: T[]) =>
+    devices.filter(device => {
+        if (device.manufacturerData.filterPolicy?.pairing === false) return true;
```

Comment:
I don't think this is what you want. In case `filterPolicy` is not present (as discussed yesterday in person), you'd continue with filtering. However, since you don't know the state, you should keep such devices.

```suggestion
        if (device.manufacturerData.filterPolicy?.pairing !== true) return true;
```